src/main/default/lwc/navigationButton/__test__/navigationButton.test.js

```diff
@@ -0,0 +1,139 @@
+import { createElement } from 'lwc';
+import navigationButton from 'c/navigationButton';
+
+function createTestComponent(props = {}) {
+  const element = createElement('c-navigationButton', {
+    is: navigationButton
+  });
+
+  document.body.appendChild(element);
+  Object.keys(props).forEach((key) => {
+    element[key] = props[key];
+  });
+
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function allPromisesResolution() {
+  // eslint-disable-next-line no-undef
+  return new Promise((resolve) => setImmediate(resolve));
+}
+
+jest.mock('@salesforce/label/c.cookbook_Next', () => ({ default: 'Next' }), {
+  virtual: true
+});
+jest.mock('@salesforce/label/c.cookbook_Back', () => ({ default: 'Back' }), {
+  virtual: true
+});
+
+describe('c-navigation-button', () => {
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it('should display the lightning button', async () => {
+    const element = createTestComponent();
+
+    await allPromisesResolution();
+    const buttonNode = element.shadowRoot.querySelector('lightning-button');
+    expect(buttonNode).not.toBeNull();
+  });
+
+  it('should display the correct label', async () => {
+    const expectedLabel = 'Next';
```

Comment:
I would suggest to use a value that differs from the default, just to "prove" that the component renders the custom label, and not the default one.

The current test would still work when omitting the `label` option, when it should fail.

    const element = createTestComponent({});